app/views/maily/shared/_stylesheet.html.erb

```diff
@@ -248,7 +264,10 @@
     text-decoration: none;
     font-size: 0.9em;
   }
-  aside.sidebar section.nav_list ul li a:hover, aside.sidebar section.nav_list ul li a.selected_mail {
+  aside.sidebar section.nav_list ul li a:hover,
+  aside.sidebar section.nav_list ul li a.selected_mail,
+  .version_list__wrapper ul.version_list li a.selected_mail,
```

Comment:
for consistency with the rest of css naming, the `__` seems a bit weird, maybe a simple `_` matches better the current style